src/selection-recommendation/recommend.py

```diff
@@ -0,0 +1,130 @@
+import sys
+sys.path.append("../spider/util")
+import itertools
+import pymysql
+import Database
+import time_conflicts_check
+
+DB_NAME = "uoftcourses"
+DB_PATH = "../../database.info"
+
+
+def get_course_data(course_code, campus):
```

Comment:
I am going to make a directory 'util' under courserecommendation to store methods like this. After that can you delete these few lines and use the method from util?